src/components/DistanceMapView/distanceMapViewPropTypes.js

```diff
@@ -2,7 +2,7 @@ import PropTypes from 'prop-types';
 
 const propTypes = {
     // Public access token to be used to fetch map data from Mapbox.
-    accessToken: PropTypes.string.isRequired,
+    accessToken: PropTypes.string,
```

Comment:
Why do we need to remove `isRequired`? and if we're removing, then shouldn't we also set some default value?

Reply:
@MaciejSWM This still seems not responded.

Reply:
Hmm I can't recall why. Seems like a mistake to me right now so I'm bringing it back. Thanks!